### DISCUSSION TITLE
Vercel + nuxt-image

### DISCUSSION THREAD

**Discussion by Yegaston** [nuxt/image#589] [General]
Hi!
I am struggling with the following problem, with nuxt-image + vercel deploys in nuxt 3 latest

Following the steps of the official docs of nuxt-img, when gonna to deploy into vercel, the images goes into 404.

```
# nuxt.config.ts
import { defineNuxtConfig } from 'nuxt';
export default defineNuxtConfig({ modules: ['@nuxt/image-edge'] });
```

```
# app.vue
<template>
  <div>
    <nuxt-img src="/nuxt-icon.png" />
  </div>
</template>
```
someone have a idea how make this work?

ty

**Comment by danielroe** [comment]
Where is the file located?

**Reply by Yegaston** [reply]
/public/nuxt-icon.png

**Reply by cpreston321** [reply]
@Yegaston I have the same problem.

**Reply by Jiangmenghao** [reply]
I have the same problem.

**Reply by Yegaston** [reply]
Yeah, is a bug with the vercel builder of [Nitro](https://nitro.unjs.io/deploy/providers/vercel), due a changes of how vercel handle the output in build time of the server side, seems nitro is not put the references to the images, in the config.json file ([Reference](https://github.com/nuxt/image/discussions/548#discussioncomment-3175247))
I have understood that the team has knowledge of this and should be fixed in the final release

Meanwhile, my workaround with this trouble was use cloudinary as image cdn and config that in a way that works when vercel being supported without any change in the code

**Reply by YusufcanY** [reply]
It seems they haven't fixed this issue.
